components/Post/PostContent.tsx

```diff
@@ -31,11 +36,22 @@ const PostContent = ({ post }: Props) => {
       </Head>
 
       <div className="blogpost-image">
+        {isPostADraft(post) && <DraftPostMark />}
+        {isPostInTheFuture(post) && <FuturePostMark />}
+
         <img
-          src={post.featuredImage ?? "/post-images/placeholder.png"}
+          src={
+            post.featuredImage ? post.featuredImage : "/post-images/draft.webp"
+          }
           alt="blog post image"
           width={790}
           height={394}
+          style={
+            isPostADraft(post) || isPostInTheFuture(post)
+              ? { filter: "grayscale(50%)", borderRadius: "3px" }
+              : //Added borderRadius, because it haven't been working
```

Comment:
Remove comment

Reply:
What this like mean?

You like some of those comments, and then just skip some of the comments

Reply:
I like it for myself. It's like checklist, so I'll know, that I answered to the each of your comments